README.md

```diff
@@ -67,37 +88,42 @@ public enum NSExceptionCode {
 
 ## Build & test
 
-Note: if you don't wish to install Gradle you can use it with wrapper: `./gradlew` instead of `gradle`.
+> Note: if you don't wish to install Gradle you can use it with wrapper: `./gradlew` instead of `gradle`.
 
-- Configure `TESTING_PROVIDER_URL` env variable with your blockchain provider for testing;
-- Run builds with `gradle build` (implies run of tests);
-- `gradle build -x test` - build without running tests. 
+- Configure a `TESTING_PROVIDER_URL` environment variable with your blockchain provider for testing.
+- To run a build with associated tests, use `gradle build`.
+- To run a build without running the tests, use `gradle build -x test`.
 
 ## Internal network config
 
 Internal [network config](src/main/resources/com/unstoppabledomains/config/network/network-config.json) 
-can be updated by running `gradle pullNetworkConfig` task and committing updated file.
+can be updated by running the `gradle pullNetworkConfig` task and committing the updated file.
 
 ## Versioning & release process
 
-##### With the updates to the codebase:  
+### On codebase updates
 
-- [CHANGELOG.md](CHANGELOG.md) should be updated with the listed changes, and 
-a corresponding client version should be bumped in [client.json](src/main/resources/com/unstoppabledomains/client/client.json).
+- [CHANGELOG.md](CHANGELOG.md) will be updated with the listed changes, and 
```

Comment:
I think it's better to highlight here that this won't be done automatically, and the corresponding action is required from the contributor (replace 'will' with something like 'required/needed/must/etc').

Reply:
Updated 👍